packages/react/src/components/Tooltip/index.tsx

```diff
@@ -44,15 +44,14 @@ export default function Tooltip({
   target,
   association = 'aria-describedby',
   variant = 'text',
-  show: showProp = false,
+  show: initialShow = false,
   hideElementOnHidden = false,
   className,
   ...props
 }: TooltipProps) {
   const [id] = propId ? [propId] : useId(1, 'tooltip');
-  const hovering = useRef(false);
```

Comment:
Instead of trying to maintain the state of whether or not we are focussed, I changed the implementation to a "deadman switch"--if you perform a "blurring" action, we will hide the tooltip unless you perform a "hovering" action within the timeout.